Formula/eris.rb

```diff
@@ -0,0 +1,25 @@
+class Eris < Formula
+  desc "Blockchain application platform CLI"
+  homepage "https://erisindustries.com"
+  url "https://github.com/eris-ltd/eris-cli/archive/v0.11.4.tar.gz"
+  sha256 "e2eb02d01b76e8be9f28aac31b2e56ebadc4d0decb21fcfefb2f219cb03d1238"
+
+  bottle :unneeded
```

Comment:
Just remove the `bottle :unneeded` line. Our CI will automagically add bottles if/when merged.